### ISSUE TITLE
The role of Social Purpose Corporations

### ISSUE THREAD

**Issue by Beanow** [sfosc/sfosc#76]
@kylerankin brought up Social Purpose Corporations in #48 and an example of such a company: https://puri.sm

In the PR the description is:

> ### Social Purpose Company
>
> An up-and-coming business model is competing head-to-head with regular technology offerings, but enshrining into the articles of incorporation the beliefs and philosophy of the free software movement. Purism, SPC. is the best example of this legal structure, a [Social Purpose Company](https://puri.sm/about/social-purpose/). This model allows for a company to sell competing products, hardware, software, and even services, that are equivalent to proprietary company offerings, but are rooted in the unwavering social values we free software supporters want. 
>
> This model allows for all funds to be used to advance social good in technology, and have large enough revenues to compete on an even and fair playing field.
>
> If the SPC forms with a legal filing that lists "all software will be released under a free software license" then you can rest-assured that all code will be released under an acceptable license, as it becomes a legal requirement for the SPC to follow its articles of incorporation.

@kylerankin from what I understand you're also on the Purism team, I hope you'll be around to share your insights!

Some things that peaked my curiosity so far
- [How Purism Works Upstream and Gives Back](https://puri.sm/posts/how-purism-works-upstream-and-gives-back/)
It explains there's a need to rebrand and make changes in a fork, but emphasizes it's not a break with the community and there's commitment to working with upstream.
- [Social purpose](https://puri.sm/about/social-purpose/) looks like a legally binding contract to prioritize the FOSS values over maximizing profits. I wonder how enforceable this is and what the limits are. If bankruptcy is imminent, can you fall back to profits over FOSS? Does something like this exist outside the US? How do you resolve a conflict, do you sue?

The big burning questions I have now though are,

- How can communities leverage a SPC to become sustainable free and open source communities?
- For what kind of communities does and doesn't this make sense?
- Pros and cons compared to Foundations and for-profits?
- Any unique pitfalls to be aware of?

**Comment by kylerankin** [comment]
> Some things that peaked my curiosity so far
>

Let me start by explaining that the SPC is a corporate charter, like being a traditional C-Corp or LLC in the US, but allows a company to put its Social Purpose into this corporate charter and be restricted by them. We are one of the first (if not the first) to apply this corporate structure to a technology company like ours. It's more typical to see clean energy companies or organic food companies use this structure because they want to protect themselves from being sued if an investor were to come along and want those companies to make more profit by, respectively, using fossil fuels or using cheaper non-organic foods.

I should also make clear that I'm not an attorney so these replies are based on my limited understanding of the corporate structure.

> How Purism Works Upstream and Gives Back
> It explains there's a need to rebrand and make changes in a fork, but emphasizes it's not a break with > the community and there's commitment to working with upstream.

Yes, our particular SPC charter requires that we use FOSS, and yes, we work with upstreams and submit PRs when we change upstream code in our own products. You can see that in our general work with Debian for our PureOS project, and with Heads for our PureBoot project. In the case of our Librem One services, we needed to rebrand for the convenience and simplification reasons outlined in that blog post.

> Social purpose looks like a legally binding contract to prioritize the FOSS values over maximizing profits. I wonder how enforceable this is and what the limits are. If bankruptcy is imminent, can you fall back to profits over FOSS? Does something like this exist outside the US? How do you resolve a conflict, do you sue?

It allows you to create a legally-binding corporate charter that defines your company's "Social Purpose" which could be "use renewable energy" or "use organic food" or "use free software" all depending on whatever social good your company wants to support. This Social Purpose takes priority over making profit. 

In our case we wanted to be able to take outside investment without being concerned (like you are with a traditional C-Corp in the US) with being sued by an investor for not maximizing shareholder value by selling proprietary software, adding spyware, locking users in, or any of the other traditional ways that hardware/software/services companies make money.

> 
> 
> The big burning questions I have now though are,
> 
> How can communities leverage a SPC to become sustainable free and open source communities?

The SPC model is intended to be used for companies that would otherwise incorporate as an LLC or traditional C corp. By incorporating in this way, the company is protected from future investors who may want to steer the company away from its Social Purpose (in our case, free software and user privacy).

> For what kind of communities does and doesn't this make sense?

This only makes sense for communities who intend on forming a corporation and want to protect themselves from investors/shareholders steering the company away from free software to make a profit. If the community isn't a corporation then SPC doesn't make sense.

> Pros and cons compared to Foundations and for-profits?

I'm not an expert on foundations, but SPC would be somewhat similar to a non-profit in the sense that your company exists for a social good, but the company would be allowed to make a profit and not have any limit on the profit it can make.

The benefit over a traditional for-profit company is that at least in the US, a traditional C-Corp can be (and has been) sued by shareholders if they do not maximize shareholder value. So a FOSS company who files as a C-Corp, then takes outside investment, risks a lawsuit if they turn down opportunities that would violate their FOSS ethics.

In our case, as a company that sells bundled hardware/software/services, much like Apple, we could, for instance, be sued as a C-Corp, by an investor, if they wanted us to bundle in a 3rd party adware into our laptops, because that 3rd party would pay us a hefty amount of money per-laptop to include it. Because we are an SPC, we can point to our Social Purpose charter and make it clear that we must put that Social Purpose above profit.

> Any unique pitfalls to be aware of?

The corporate structure is still relatively new, so I don't know how many legal challenges have been tested in court yet. As far as funding goes, our SPC structure, and the ethics we have in our corporate charter, has discouraged a certain type of venture capitalist, who is looking for quick profits, from investing in us, because they see that we are looking, instead, for a long-term social good.

But, for instance, if you were an Open Core company that filed as an SPC, you might have a problem if you wanted to pivot over to the Business Public License, if you put in your SPC charter that you would always use free software licenses for your code. I believe in that case the company would have to file as a new C-Corp and close the SPC business, which would require the consent of the majority shaerholders. But then, an Open Core company couldn't put that kind of free software clause in their corporate charter, and sell any kind of proprietary module/plugin, either, so it's a moot point.

**Comment by Beanow** [comment]
@kylerankin thank you for following up :smile: That is a really comprehensible way to put it.
And gave me an idea of why you might as a community prefer a SPC vs a non profit. You still do have pressure to turn a profit with your products / services (though not above your embedded social purpose), so it could help push foss projects to a finished product level of refinement, in a way an NPO might not. Would you agree with that line of thought?

Though I do wonder, does a SPC additionally open itself up to being sued over not adhering to it's social purpose? Like your example, if an SPC that enshrined foss and privacy did go for that spyware after all (from within the company or with informal shareholder pressure), could a contributor sue because it's a breach with your social purpose?
Or what about not _maximising_ the social purpose? For example if there's 30 FTE worth of payout going to investors and a growing backlog of things to do to give back upstream, could you sue for not hiring more devs with these profits?

In other words, besides protection from being sued to give up your social purpose for profits, does it give new guarantees you will prioritize the social purpose?
